.ci_support/linux_64_.yaml

```diff
@@ -5,4 +5,6 @@ channel_sources:
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-cos7-x86_64
+- quay.io/condaforge/linux-anvil-x86_64:alma9
+python_min:
+- '3.9'
```

Comment:
> @MetRonnie why not merge this one?

We'll set `python_min` in a `recipe/conda_build_config.yaml`  file, set the value to 3.7 and rerender so that these CI scripts pick up the correct value.

Reply:
> Note that Python 3.7 is no longer supported in conda-forge, and else where too. Do that at your own risk.
BTW, you don't need a conda_build_config.yaml

We still support it, and 3.8.

>  Just add {% set python_min = "3.7" %} at the top of the recipe to override it.

We found that the python_min value in this file was not set correctly when we tried this approach on another feedstock.

Reply:
> We found that the python_min value in this file was not set correctly when we tried this approach on another feedstock.

If you are expecting the global min to change, that won't happen. But the version set there will work. If not, please report a bug.

Reply:
> We still support 3.7, and 3.8.

Sure. What I said is that the 3.7 version of Python itself in conda-forge is not supported. You will be using something without update/security patches.

Reply:
I am well aware of that, although note, pypy and some cpython vendors still support 3.7.

Reply:
All we need is an acknowledgement so you know there won't be any support on conda-forge's side for dropped versions.